lib/commands/migrate.js

```diff
@@ -178,9 +178,15 @@ function runMigration(currentConfigPath, outputConfigPath) {
 							);
 						}
 					}
+					console.log(chalk.green(`✔︎ New webpack config file is at ${outputConfigPath}.`));
```

Comment:
We lost the new line before these messages and are now printing unnecessary newlines between each one. We should really just clean this up and add these statements into a single `console.log` with a template string (this avoids four unnecessary function calls):

```js
console.log(chalk.green(`
✔︎ New webpack config file is at ${outputConfigPath}.
✔︎ Updating to the latest version can contain breaking changes.
✔︎ Plugin and loader dependencies may also need to be updated.
`));
```

Alternatively we can keep them separate like they are now:

```js
console.log(chalk.green(`\n✔ New webpack config file is at ${outputConfigPath}.`));
console.log(chalk.green("✔ Updating to the latest version can contain breaking changes."));
console.log(chalk.green("✔ Plugin and loader dependencies may also need to be updated."));
```